examples/step-7/step-7.cc

```diff
@@ -967,15 +967,15 @@ namespace Step7
       {
         if (cycle == 0)
           {
-            GridGenerator::hyper_cube(triangulation, -1, 1);
+            GridGenerator::hyper_cube(triangulation, -1., 1.);
             triangulation.refine_global(3);
 
             for (const auto &cell : triangulation.cell_iterators())
               for (const auto &face : cell->face_iterators())
                 {
                   const auto center = face->center();
-                  if ((std::fabs(center(0) - (-1)) < 1e-12) ||
-                      (std::fabs(center(1) - (-1)) < 1e-12))
+                  if ((std::fabs(center(0) - (-1.0)) < 1e-12) ||
```

Comment:
The other places are not that important, but here I think it helps the reader make it obvious what it is.

Reply:
Yes, good point.